lib/screens/dji_shop/screens/dji_shop_screen/widgets/product_card.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_animation/infrastructure/theme/app_colors.dart';
+import 'package:flutter_animation/infrastructure/theme/app_images.dart';
+import 'package:flutter_animation/infrastructure/theme/theme_extension.dart';
+import 'package:flutter_animation/screens/dji_shop/screens/product_screen/product_screen.dart';
+
+// ---Parameters---
+const _kPadding = 15.0;
+const _kWidthHeight = 300.0;
+
+class ProductCard extends StatelessWidget {
+  const ProductCard({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
```

Comment:
you still not using `PageView` widget
pls replace `SingleChildScrollView` to `PageView`